public/config.json

```diff
@@ -1,4 +1,4 @@
 {
 	"stage": "dev",
-	"endPoint": "https://digu74hvnk.execute-api.us-east-1.amazonaws.com/dev"
+	"endPoint": "https://digu74hvnk.execute-api.us-east-1.amazonaws.com/dev/"
```

Comment:
Please change the name of the file to config_mt_intg_01_aws_us_east_1.json per our Environments documents.
In the future, we'd like to change this to eScribers.net domain with the exact environment name as part of the URL.
What do we need the stage for?
Shall we change it or add "environment": "mt-intg-01-aws-us-east-1"?